creator-node/src/services/stateMachineManager/stateMonitoring/fetchCNodeEndpointToSpIdMap.jobProcessor.ts

```diff
@@ -32,6 +33,7 @@ module.exports = async function ({
       audiusLibs.ethContracts
     )
   } catch (e: any) {
+    tracing.recordException(e)
```

Comment:
in order to see exceptions in traces do we have to add this `recordException` in every catch?

Reply:
In order to see exceptions that are caught and handled, yes. It's the same as logging an error

Uncaught exceptions will automatically be logged by the decorator

Reply:
cool cool